doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -444,10 +444,13 @@ not its name, one usually uses “arrow” abstractions for prenex
 arguments, or the ``(_ : term)`` syntax for inner arguments. In |SSR|,
 the latter can be replaced by the open syntax ``of term`` or
 (equivalently) ``& term``, which are both syntactically equivalent to a
-``(_ : term)`` expression.
+``(_ : term)`` expression:
+
+.. prodn::
+   binder += binder | & term | of term
 
 For instance, the usual two-constructor polymorphic type list, i.e.
-the one of the standard List library, can be defined by the following
+the one of the standard `List` library, can be defined by the following
```

Comment:
To have a verbatim rendering you need two backticks before and after: ``` ``List`` ```

Reply:
Thanks. Side remark: references to libraries are very heterogeneous (in particular, with or with .v). Is there a general policy?